index.bs

```diff
@@ -168,46 +173,43 @@ Example:
-This means bi-directional communication between the player and creative
-requires using the postMessage API. This, in turn, means that all
-communication between player and creative is asynchronous.
-
 ## SIMID Ad Serving Flow ## {#video-ad-flow}
 
-The SIMID ad experience is delivered by concurrently rendering
-the streaming media and the interactive creative. The player obtains
-both media and creative data from VAST 4.x, loads the resources, assembles
-the ad, and ensures a cohesive ad experience.
+The SIMID ad experience is delivered by a web browser or application concurrently rendering
+an ad's streaming audio or video file and its interactive creative file. The media player obtains
+urls for both files from a VAST document, loads the files, assembles
+them together into a single ad unit, and ensures a cohesive ad experience.
 
-Diagram below illustrates resources loading and ad compilation.
+This diagram illustrates the SIMID creative loading and presentation process.
 <img width="100%" src="images/simid_diagram_4_loading.png" alt="Diagram illustrates resources loading and ad compilation.">
 
 ## Player and Creative Communication ## {#player-creative-communication}
 
-Player and SIMID creative communicate by sending bidirectional messages over
-the standard postMessage API. SIMID API requirements govern message construction
-conventions as well as the message data structure.
+A media player and a SIMID creative communicate by sending serialized messages back and forth to each other.
+
+Because a SIMID creative is an HTML document that is served from an advertiser’s
+web domain, and it is loaded by a media player into an iframe within a web page hosted on a different domain,
+loading the creative requires the creation of a cross-origin iframe (also known as an "unfriendly iframe").
+Due to browser sandbox security restrictions, javascript communication across this type of iframe can only be achieved via the standard postMessage API.
 
-See sections [[#msg-proto]], [[#api]].
+SIMID API requirements govern message construction conventions as well as the message data structure. See sections [[#msg-proto]], [[#api]] for more information.
 
 ## Scope and Limitation ## {#scope}
 
 The use of HTML is only required for the SIMID creative, not the publisher
 property hosting that creative. As long as the publisher can load HTML and
-communicate with it over the standard postMessage interface, it can support
+communicate with it over the standard postMessage API, it can support
 SIMID. In practice, this means that SIMID can be hosted in web page iframes,
-mobile app web views, and other platforms. In fact, SIMID can better support mobile use cases since SIMID can be used with the native player.
+mobile app web views, and other platforms. In fact, SIMID can better support mobile use cases than VPAID because SIMID has the ability to communicate with the native player via postMessage.
 
```

Comment:
Editor's Note: I'm not actually certain me above edit is technically correct: `... SIMID has the ability to communicate with the native player via postMessage.`  I was trying to clarify the prior version to be more clear, and assuming SIMID actually _can_ communicate with a native device player I figured it _has_ to be via postMessage, right? Please correct me if I'm wrong, which might mean that entire sentence should be removed?

Reply:
I'm also not sure if that's technically correct. This needs an engineer to confirm.

Reply:
It is, but it's not about the communication per se. SIMID is designed to allow the video player to be in the host layer, which means native in apps, whereas VPAID put the responsibility to manage the video player into the creative. (Though even in that case it can be pushed into native layer, but it wasn't designed with that in mind.)

I wasn't sure how to write this in a more spec-proper language, so I didn't make a suggestion here. :)

Reply:
OK, I rewrote this to focus on the control of the media asset (rather than the communication). Let me know if you think this works better:
`In fact, SIMID can better support mobile use cases than VPAID because a native app or media player directly controls loading and playback of a SIMID ad unit's
media asset (whereas a VPAID ad unit offers no direct access to or control of its internal media asset).`